fedimint-ui/apps/guardian-ui/package.json

```diff
@@ -3,13 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "react-scripts start",
-    "dev": "react-scripts start",
+    "start": "yarn isNix && react-scripts start",
```

Comment:
Two objections with `isNix` check.

- If we have yarn available, do we still need enforce nix shell with `isNix` check?! At present, we only use nix to provide yarn as a standard dev tool, so we don't require ui devs to install yarn globally before they can start the UIs.

- If we still want to enforce nix shell, we should be careful to do so in dev only, but not in prod scenarios

Reply:
It might be related to my comment here: https://github.com/fedimint/fedimint/pull/2518#pullrequestreview-1426268946

I agree we shouldn't just refuse to work if all the tooling is present either way, it would still be nice to point non-ui devs that just want to test if they broke anything the right way. It's quite hard to figure out how to run `just run-ui` right now, it doesn't *just work* :laughing:

Reply:
Yes, I had the idea from review #2518.

>  If we have yarn available, do we still need enforce nix shell with isNix check?! At present, we only use nix to provide yarn as a standard dev tool, so we don't require ui devs to install yarn globally before they can start the UIs.

`isNix` does not enforce anything. It checks for the ENV VAR and outputs a message if it's not present. I'm happy to remove it if that is conflicting but there is no enforcement of nix with this.

> If we still want to enforce nix shell, we should be careful to do so in dev only, but not in prod scenarios

Similar to previous, but this leads to another question around how we envision these web apps being deployed. I don't believe we would want to run npm commands to deploy? Usually, you run [build commands to create a static site and then serve from a web server](https://create-react-app.dev/docs/deployment/). `yarn build` is unaffected by `isNix`.

Reply:
> `isNix` does not enforce anything.

That sounds like exactly what we want then!